cmd/syncv3/main.go

```diff
@@ -113,10 +120,53 @@ func main() {
 		AddPrometheusMetrics: args[EnvPrometheus] != "",
 	})
 
+	if args[EnvSentryDsn] != "" {
+		fmt.Printf("Configuring Sentry reporter...\n")
+		err := sentry.Init(sentry.ClientOptions{
+			Dsn:     args[EnvSentryDsn],
+			Release: version,
+			Dist:    GitCommit,
+		})
+		if err != nil {
+			panic(err)
+		}
+	}
+
 	go h2.StartV2Pollers()
 	if args[EnvJaeger] != "" {
 		h3 = otelhttp.NewHandler(h3, "Sync")
 	}
+
+	if args[EnvSentryDsn] != "" {
+		defer sentry.Flush(2 * time.Second)
```

Comment:
We surely don't need to flush here as we flushed in `WaitForShutdown`?

Reply:
Err, yes. I think I did a sloppy revert.